### ISSUE TITLE
Workbox configuration

### ISSUE THREAD

**Issue by VuongVu** [hanford/next-offline#143]
After build i see a notice: 
```
Compiled with warnings.

You're using the following Workbox configuration options: [globDirectory, globPatterns]. In Workbox v3 and later, this is usually not needed. Please see https://goo.gl/EQ4Rhm for more info.
```
Does this mean my configuration is not valid in the new version?
This is my config: 
```
workboxOpts: {
    swDest: 'static/service-worker.js',
    clientsClaim: true,
    skipWaiting: true,
    cleanupOutdatedCaches: true,
    runtimeCaching: [
      {
        urlPattern: /^https?.*/,
        handler: 'networkFirst',
        options: {
          cacheName: 'https-calls',
          networkTimeoutSeconds: 15,
          expiration: {
            maxEntries: 150,
            maxAgeSeconds: 30 * 24 * 60 * 60, // 1 month
          },
          cacheableResponse: {
            statuses: [0, 200],
          },
        },
      },
    ],
  }
```
I have another question, does sw automatically recognize the new version after I deploy my server again, otherwise how to do it? Thank you.

**Comment by hanford** [comment]
Going to close this issue, I think it's a duplicate of this: https://github.com/hanford/next-offline/issues/117

tl;dr the settings are being deprecated but they're not fully deprecated yet.

the SW _will_ automatically recognize newer versions. you're using a `networkFirst` cache strategy which means next-offline will only fall back to the cache if the network isn't available.
